.travis.yml

```diff
@@ -23,17 +41,18 @@ before_install:
       brew install mumps;
     fi
   - if [ `uname` == "Darwin" ]; then export TMPDIR=/tmp; fi  # for OpenMPI
+  - export CC=mpicc;
+  - export FC=mpif90;
   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
+  - rm -rf $HOME/.julia/registries
 
 install:
-  - julia -e 'versioninfo()'
-  - julia -E 'Pkg.clone(pwd());'
-  - julia -E 'Pkg.build("MUMPS");'
-  - julia -E 'using BinDeps; BinDeps.debug("MUMPS")'
+  - julia -e 'using InteractiveUtils; versioninfo()'
+  - julia -E 'using Pkg; Pkg.clone(pwd()); Pkg.build("MUMPS"); Pkg.test("MPI");'
```

Comment:
Why are you saying `Pkg.test("MPI")`, if you want to do that you have to do a `Pkg.add("MPI")` first.

Reply:
MPI is already in my REQUIRE file. I added test to debug issues while building on CI.

Reply:
It is in your `REQUIRE` file and thereby in your `Manifest.toml`, but it is not in your `Project.toml`. At least that's what the error says. You can test this locally

```
julia --project=newenv
> using Pkg
> Pkg.add("MUMPS#julia-0.7-initial")
> Pkg.status()
> using MPI  # this will fail
```